routers/web/repo/pull.go

```diff
@@ -745,7 +744,7 @@ func UpdatePullRequest(ctx *context.Context) {
 	// ToDo: add check if maintainers are allowed to change branch ... (need migration & co)
 	if (!allowedUpdateByMerge && !rebase) || (rebase && !allowedUpdateByRebase) {
 		ctx.Flash.Error(ctx.Tr("repo.pulls.update_not_allowed"))
-		ctx.Redirect(ctx.Repo.RepoLink + "/pulls/" + fmt.Sprint(issue.Index))
+		ctx.Redirect(ctx.Repo.RepoLink + "/pulls/" + url.PathEscape(fmt.Sprint(issue.Index)))
```

Comment:
Is it `ctx.Redirect(issue.Link())`?

Reply:
Does issue definitely have the repo loaded here?

Reply:
From this line to next `issue.Link()` (around line 767), there is no more update to `issue`.

So I think calling `issue.Link()` here is the same as next `issue.Link()`

Reply:
```suggestion
		ctx.Redirect(issue.Link())
```